src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:3000/api" method="post"
+    name="testform">
```

Comment:
```suggestion
    <form
      action="http://localhost:3000/api"
      method="post"
      name="testform"
    >
```
fix your code style
check this checklist point
<img width="929" alt="image" src="https://user-images.githubusercontent.com/102411130/230554647-a5fae825-164c-49b4-af1a-a2e78963985c.png">